src/mqt/bench/evaluation/evaluation.py

```diff
@@ -0,0 +1,61 @@
+from __future__ import annotations
+
+import pickle
+import sys
+from dataclasses import dataclass
+from pathlib import Path
+from typing import TYPE_CHECKING
+
+from joblib import Parallel, delayed
+from mqt.bench import utils
+from qiskit import QuantumCircuit
+
+if TYPE_CHECKING or sys.version_info >= (3, 10, 0):  # pragma: no cover
+    from importlib import resources
+else:
+    import importlib_resources as resources
+
+
+def create_statistics() -> None:
+    source_circuits_list = [file for file in Path(utils.get_qasm_output_path()).iterdir() if file.suffix == ".qasm"]
+    res_dicts = Parallel(n_jobs=-1, verbose=100)(
+        delayed(evaluate_qasm_file)(str(filename)) for filename in source_circuits_list
+    )
+    target_dir = Path(resources.files("mqt.bench") / "evaluation/")
+    with Path(target_dir / "evaluation_data.pkl").open("wb") as f:
```

Comment:
```suggestion
    with (resources.files("mqt.bench") / "evaluation" / "evaluation_data.pkl").open("wb") as f:
```
Should also work and be more compact.